README.md

```diff
@@ -255,7 +254,19 @@ Some macros you can use:
 
 Trying to compile the empty string raises an error (because this is more often a mistake than not). In the rare case you need it, use `[]`.
 
-Coming soon: `#integer`, `#ip`, ..., `#a..f`, `abc[ignore_case 'de' #lowercase]` (which translates to `abc[['D' | 'd'] ['E'|'e'] [[A-Z] | [a-z]]`, today you just wouldn't try), `[#0..255]` (which translates to `['25' #0..5 | '2' #0..4 #d | '1' #d #d | #1..9 #d | #d]`, `[capture:name ...]`, `[1+:fewest ...]` (for non-greedy repeat), unicode support. Full PCRE feature support (lookahead/lookback, some other stuff). See TODO.txt.
+Coming soon:
+
+- `#integer`, `#ip`, ..., `#a..f`
+- `numbers: #number_scientific`, `#decimal` (instead of `#real`)
+- improve readability insice brackets scope with `#dot`, `#hash`, `#tilde`...
+- `abc[ignore_case 'de' #lowercase]` (which translates to `abc[['D' | 'd'] ['E'|'e'] [[A-Z] | [a-z]]`, today you just wouldn't try)
+- `[#0..255]` (which translates to `['25' #0..5 | '2' #0..4 #d | '1' #d #d | #1..9 #d | #d]`
+- `[capture:name ...]`, `[1+:fewest ...]` (for non-greedy repeat)
+- unicode support. Full PCRE feature support (lookahead/lookback, some other stuff)
+- Option to add your macros permanently. `ke.add_macro("#camelcase=[1+ [#uppercase [0+ lowercase]]], path_optional)`, `[add_macro #month=['january', 'January', 'Jan', ....]]`
```

Comment:
Maybe "globally" instead of "permanently"? Because it's just for the running process :)